mrp_subcontracting_partner_management/__manifest__.py

```diff
@@ -6,7 +6,8 @@
     "license": "LGPL-3",
     "category": "Inventory",
     "website": "https://github.com/OCA/manufacture",
-    "depends": ["purchase_stock", "mrp_subcontracting", "sale_stock"],
+    "depends": ["purchase_stock", "mrp_subcontracting_purchase", "sale_stock"],
```

Comment:
You shouldn't change dependencies in a stable module. Do instead an extension module.